src/Utils.h

```diff
@@ -24,11 +24,11 @@
 /*
 Custom functions define here so they can be compiled on both x64 and ARM
 */
-#define jankabs(x) ((x>0)-(x<0))*x
+#define jankabs(x) (((x)>0)-((x)<0))*(x)
 
 template<class T> const T pasta_abs(const T x)
 {
-    return ((x>0)-(x<0))*x;
+    return (((x)>0)-((x)<0))*(x);
```

Comment:
What's the difference between `pasta_abs` and `jankabs`? Can we eliminate one or add a comment explaining why we'd use one over the other?

Reply:
we can eliminate jankabs. I used pasta_abs only bc I once got differing behavour from the ardionu fabs and the cpp standard library fabs, and fundamentally I don't like the idea of having a different function called in hootl vs on the arm.

Reply:
Dope